calculator.rb

```diff
@@ -0,0 +1,92 @@
+# The program adds numbers with both add and +
+# The program subtracts numbers with both subtract and -
+# The program adds numbers with both multiply and *
+# The program adds numbers with both divide and /
+# The program handles divide when attempting to divide by zero.
+# The program handles erroneous input. For example the user might enter clown when asked to enter a number.
+# The program also needs to handle erroneous operators.
+
+# Print out the formula in addition to the result, i.e. 2 + 2 = 4
+# Make your program know when it needs to return an integer versus a float.
+# Add support for computing exponents (2^4 = 2 * 2 * 2 * 2 = 16).
+
+# method created to check user input, if it has decimal store as float
+def number
+    number = number.to_s
```

Comment:
Your indentation is consistent throughout, but it looks like you're using 4 spaces to indent, while our standard at Ada is to use 2. There's not a huge advantage to one or the other, but it's important that we all do the same thing, especially once we start getting into pair projects.

Please adjust your editor settings to use 2-space indentation.